resources/keywords/dbaas_policy.resource

```diff
@@ -39,25 +42,32 @@ Create Custom Namespace
     ${lblProjectSelectUpd}=     Replace String    ${lblProjectSelect}    <<project>>    ${custom_ns}
     Wait Until Page Contains Element    ${lblProjectSelectUpd}
 
-Setup DBaaSPolicy for ${namespace}
+Setup DBaaSPolicy for ${namespace} as ${DBaaSPersona}
```

Comment:
This is adding the logged in persona in addition to the namespace right?

Reply:
@ikanias This keyword would create a DBaaSPolicy for a namespace with given DBaaSPersona like Project Admin, Service Admin and Developer. So the cluster is already logged in with the DBaaSPersona mentioned.